roles/update/tasks/create-custom-openstackclient.yaml

```diff
@@ -0,0 +1,23 @@
+---
+# The default openstackclient might be restarted during update.
+# Create a copy to avoid any restart.
+- name: Copy parameter from the existing openstackclient Pod
+  kubernetes.core.k8s_info:
+    kubeconfig: "{{ cifmw_openshift_kubeconfig }}"
+    api_key: "{{ cifmw_openshift_token | default(omit) }}"
+    context: "{{ cifmw_openshift_context | default(omit) }}"
+    namespace: "{{ cifmw_install_yamls_defaults['NAMESPACE'] }}"
+    kind: Pod
+    name: "openstackclient"
```

Comment:
We need to Openstack controlplane to be deployed to be able to copy params from existing openstackclient pod, right?

Reply:
This is a wrong rebase from me at some point.  I have removed that at some point.  The idea was to circumvent the restart of the container by having another pod whose definition wouldn't be altered.

Bottom line, is that restarting container shouldn't happen at that stage in any case and this "workaround" isn't needed.

This is going to be removed.